src/cortex_cli/cortex_cli.py

```diff
@@ -984,11 +989,11 @@ def run(  # pylint: disable=too-many-arguments, too-many-locals, import-outside-
 
     if output == OutputFormat.FREQUENCIES:
         output_string = f'Circuit "{input_circuit.name}" results using '
-        output_string += f'calibration set {results.metadata.calibration_set_id} over {shots} shots:\n'
+        output_string += f'calibration set {results.metadata.request.calibration_set_id} over {shots} shots:\n'
```

Comment:
Shouldn't this and line 996 rather use `results.metadata.calibration_set_id` which is the ID that was actually used, instead of `results.metadata.request.calibration_set_id` which is the ID that was requested and that can be None.